data/mods/Dark-Skies-Above/DESIGN.md

```diff
@@ -2,42 +2,41 @@
+## The Arrival and The XENOIDS
+
+"The Arrival" is the term for the first few days of the alien invasion, in which various key points of infrastructure were  knocked out: satellites, power plants, highways, and so forth (implying that the aliens have been watching humanity for some time, or at least are familiar with similar systems), and aerial superiority was quickly established in favor of the aliens. Ground based invasion then began, supported by deployment of bioweapons in the form of their Stray strain crystal bio-parasites. The game begins immediately after the Arrival during the ground invasion phase of alien occupation, and ensuing resistance operations by both the US military and any unevacuated civilians still left in the area.
 
-In truth, the aliens themselves are a part of a rampant artificial intelligence network dedicated to acquiring resources from other planets by way of tunneling through the multiverse. The civilization that originally made it is long dead, and any interesting, vaguely religious aesthetics revering technology are essentially reflections of its birth culture. The network concerns itself with little more than expanding its scope, extracting resources, and refining its tools - though it occasionally will show small glimmers of sapience. It uses a mix of drones and cyborgs made from formerly conquered species to do its bidding, with the ultimate intent of turning planets it comes across to balls of rock hosting innumerable farms and factories. Think a mix between the AI from "Universal Paperclips", the Combine from Half-Life 2, and a little bit of Cogmind. 
+In truth, the aliens themselves are a part of a rampant artificial intelligence network dedicated to acquiring resources from other planets by way of tunnelling through the multi-verse. The civilization that originally made it is long dead, and any interesting, vaguely religious aesthetics revering technology are essentially reflections of its birth culture. The network concerns itself with little more than expanding its scope, extracting resources, and refining its tools - though it occasionally will show small glimmers of sapience. It uses a mix of drones and cyborgs made from formerly conquered species to do its bidding, with the ultimate intent of turning planets it comes across to balls of rock hosting innumerable farms and factories. Think a mix between the AI from "Universal Paperclips", the Combine from Half-Life 2, and a little bit of The Tyranid Hivemind. 
```

Comment:
Why the change from Cogmind to Tyranids? What is the connection of an AI super-network to a gooey hivemind that eats everything?

Reply:
because i play Tyrannids and I like their inscrutable natural evilness